src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.h

```diff
@@ -77,13 +77,18 @@ class BtreeLBAMapping : public BtreeNodeMapping<laddr_t, paddr_t> {
       intermediate_key(indirect ? val.pladdr.get_laddr() : L_ADDR_NULL),
       intermediate_length(indirect ? val.len : 0),
       raw_val(val.pladdr),
-      map_val(val)
+      map_val(val),
+      readable(!indirect)
```

Comment:
The reason why "readable" is set to "!indirect" is because it's only when the mapping is created by `LBAManager::clone_extent()` that `indirect` is true.

I have to admit that this is really confusing, I'll improve this part.:-)

Reply:
I don't see `BtreeLBAMapping::_get_original_mapping()`, do you mean `_get_original_mappings()`?

Looking at `BtreeLBAManager::get_mapping()`, if the queried offset is an indirect mapping, it will always query the direct mapping, so that the user will always see a "readable" indirect pin from this interface, right?